src/sentry/api/exceptions.py

```diff
@@ -85,6 +85,11 @@ class SuperuserRequired(SentryAPIException):
     message = "You need to re-authenticate for superuser."
 
 
+class DataSecrecy(SentryAPIException):
```

Comment:
Should this end in “Exception”? Can we add a comment describing what this is for?